website/docs/reference/widgets/chart.mdx

```diff
@@ -0,0 +1,412 @@
+
+For example, if you have data on the number of males and females who voted in each year of a political election, you can use the Chart Series property to display both sets of data on the same chart. This would allow you to compare the number of male and female voters over time.
+
+Check this [sample app](https://app.appsmith.com/applications/620b668f6b4b1e154a3c2418/pages/620b8bc56b4b1e154a3c2688) to learn how to produce a dual-axis chart combining columns, lines, and areas.
+
+## Perform actions based on data point selection
+
+Performing actions based on data point selection refers to the ability to execute certain actions or operations when a user clicks on a specific data point in a chart widget. 
+
+To achieve this, you can set up the **"onDataPointClick"** event to perform a designated action. For instance, you can use a Modal widget to show the details of the clicked data point using `{{Chart1.selectedDataPoint.x}}` or you can execute a query based on the user's selection.
+
+## Custom charts
+
+Appsmith offers integration with [FusionCharts](https://www.fusioncharts.com/dev/chart-guide/list-of-charts/) and provides a wide range of chart types that can be used to create custom charts. Once you have selected the Custom Chart type, you can configure the FusionCharts using the Custom Fusion Chart property.
+
+
+:::info
+FusionCharts provides an extensive range of chart variants, with almost 100+ options to choose from. To learn more about these charts, you can refer to the official [FusionCharts documentation](https://www.fusioncharts.com/dev/chart-guide/list-of-charts/) 
+:::
+
+To create a custom chart, you need to provide two keys in your configuration: type and datasource:
+
+
+* **Type**
+
+The type is used to define the chart type recognized by the FusionCharts. You can[ get a list of all the supported types available](https://www.fusioncharts.com/dev/chart-guide/list-of-charts) on FusionCharts.
+
+* **Datasource**
+
+The datasource defines the customization options and the data points to create a chart. It includes two attributes: **chart** and **data**.
+
+The chart object can customize the chart and includes options like caption, xAxisName, and yAxisName. On the other hand, the data attribute represents the data in an array format with key-value pairs like `[{“label”: “string value”, “value”: “string value”}]`.
+
+Below is an example of a configuration for the Custom Fusion Chart property:
+
+```
+{
+  "type": "",
+  "dataSource": {
+    "chart": {},
+    "data": []
+  }
+}
+```
+
+### Custom charts examples
+Here are some examples of custom charts that you can use as a reference to create your own customized charts.
+
+
+<Tabs>
```

Comment:
in the custom chart example give the entire json code so that the user can copy paste to test the example